chromium_src/chrome/common/chrome_content_client_constants.cc

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/common/chrome_content_client.h"
```

Comment:
this is actually a pretty safe place to define `GOOGLE_CHROME_BUILD`, just make sure to include `#include "chrome/common/chrome_content_client.h"` first and then all the patches go away

Reply:
Hmm, it's difficult because `kNotPresent` declaration is also in `GOOGLE_CHROME_BUILD` of `chrome_content_client.h`

Reply:
do we care about `kNotPresent`?